alsoHeresTheInterviewQuestionIHad/alsoHeresTheInterviewQuestionIHad.js

```diff
@@ -0,0 +1,43 @@
+const fs = require("fs");
+const streetArray = fs.readFileSync("data.txt", "utf-8").split(",");
+
+const streetArraySorted = streetArray.sort(function (a, b) {
+    return a - b;
+});
+
+let numberOfBuildings = streetArray.length;

```

Comment:
Since these variables are only referenced inside of our `findBestArea` function, maybe we should move them there?